redbot/cogs/warnings/warnings.py

```diff
@@ -400,13 +403,16 @@ def __init__(self, bot: Red):
                         description=user_warnings[key]["description"],
                     )
                 await ctx.send_interactive(
-                    pagify(msg, shorten_by=58), box_lang=_("Warnings for {user}").format(user=user)
+                    pagify(msg, shorten_by=58),
+                    box_lang=_("Warnings for {user}").format(user=user),
                 )
 
     @commands.command()
     @commands.guild_only()
     @checks.admin_or_permissions(ban_members=True)
-    async def unwarn(self, ctx: commands.Context, user: Union[discord.Member, int], warn_id: str):
+    async def unwarn(
+        self, ctx: commands.Context, user: Union[discord.Member, int], warn_id: str, reason: str,
```

Comment:
Reason should be optional:
```suggestion
        self,
        ctx: commands.Context,
        user: Union[discord.Member, int],
        warn_id: str,
        reason: str = None,
```

Reply:
Where is the change?

Reply:
oh i'm blind. yeah, agreed